bee-message/src/output/feature_block/indexation.rs

```diff
@@ -3,37 +3,43 @@
 
 use crate::Error;
 
-use bee_common::packable::{Packable, Read, Write};
+use bee_packable::{bounded::BoundedU8, prefix::BoxedSlicePrefix};
 
 use core::ops::RangeInclusive;
 
+pub(crate) type IndexationFeatureBlockLength =
+    BoundedU8<{ *IndexationFeatureBlock::LENGTH_RANGE.start() }, { *IndexationFeatureBlock::LENGTH_RANGE.end() }>;
+
 /// Defines an indexation tag to which the output will be indexed.
-#[derive(Clone, Debug, Eq, PartialEq, Ord, PartialOrd)]
+#[derive(Clone, Debug, Eq, PartialEq, Ord, PartialOrd, bee_packable::Packable)]
 #[cfg_attr(feature = "serde1", derive(serde::Serialize, serde::Deserialize))]
+#[packable(unpack_error = Error, with = |e| Error::InvalidIndexationFeatureBlockLength(e.into_prefix().into()))]
```

Comment:
Maybe someone else will disagree, but I'm not too keen on having closures in attributes...I think it would be better and more readable to just have this as a small function.

Reply:
I find hard to balance "closures in attributes" vs "defining a function that is only used once".